client/src/components/navbar-component/Dropdown.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import "./Dropdown.css";
+import DropdownItem from "./DropdownItem";
+
+function Dropdown({ isAdmin }) {
```

Comment:
Whenever a user clicks off of the dropdown it should close it

Reply:
Sorry I'm struggling with implementing this